docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Quickstart
+hide_title: false
+sidebar_position: 2
+---
+
+**Ready to give Apache Superset a try?** This quickstart will help you run Superset on your local machine in **5 simple steps**.
+It assumes that you have [Docker](https://www.docker.com) installed.
+
+### 1. Get Superset
+```
+$ docker pull apache/superset:3.0.2
```

Comment:
Is it possible to not pin the version, maybe something like:

```suggestion
$ docker pull apache/superset:<version>
```

Reply:
Right now it's possible but it would require versioning the documentation. The pinning is done currently to "stabilize" versions as not to let the latest tag be used, since that is currently bound to the master branch, instead of the latest release.

Reply:
Something that we could do (just an idea) would be to update the version in the docs with every release, and prioritize the latest upstream version (e.g 3.1 instead of 3.0.3)

@sfirke @michael-s-molina @rusackas would that be feasible?

Reply:
@artofcomputing the challenge is, without a 🔮, we can't be certain what the upstream version is, i.e., it could be 3.1 or 4.0.

Reply:
True, there's no way to tell. IMO we could define a semantic rule to favor the latest major version on the page, and minor/patch versions if there's no new major updates. But yeah, that would require manual bumps, or some other way, like <version> as you mentioned and let user manually set it. Any thoughts?

Reply:
That's reasonable, I'll add the placeholder with information on where to instruct users.

Reply:
@john-bodley @michael-s-molina made the change in the step 1 and 2 with commit [328e871](https://github.com/apache/superset/pull/26129/commits/328e8715016a5a5871a412dc6c6a4c703b18c9d2), and added a tip on how users can find the latest release